ansible_base/tests/unit/utils/test_authentication.py

```diff
@@ -0,0 +1,57 @@
+from unittest import mock
+
+import pytest
+from django.conf import settings
+from django.test import override_settings
+
+from ansible_base.utils.authentication import generate_ui_auth_data
+
+
+@pytest.mark.django_db
+def test_generate_ui_auth_data_no_authenticators_or_settings():
+    response = generate_ui_auth_data()
+    assert response == {
+        'login_redirect_override': '',
+        'passwords': [],
+        'show_login_form': False,
+        'ssos': [],
+    }
+
+
+@override_settings(LOGIN_REDIRECT_OVERRIDE='https://www.google.com')
+@pytest.mark.django_db
+def test_generate_ui_auth_data_valid_login_redirect():
+    response = generate_ui_auth_data()
+    assert response == {
+        'login_redirect_override': settings.LOGIN_REDIRECT_OVERRIDE,
+        'passwords': [],
+        'show_login_form': False,
+        'ssos': [],
+    }
+
+
+redirect_url = 'www.monster.com'
```

Comment:
Just because I'm paranoid about referencing company names, let's use fake URLs instead (example.com and example.org are reserved by IANA for this purpose).

```suggestion
redirect_url = 'example.com'
```

Reply:
There was also a reference to google.com that I changed to example.com.